packages/front-end/components/PowerCalculation/stats.ts

```diff
@@ -130,39 +160,34 @@ export function sequentialPowerStandardError(
  * @returns Estimated power.
  */
 export function powerEst(
-  effectSize: number,
-  mean: number,
-  variance: number,
+  metric: MetricParamsFrequentist,
   n: number,
   nVariations: number,
   alpha: number = 0.05,
   twoTailed: boolean = true,
-  sequentialTuningParameter = 0
+  sequentialTesting: false | number
```

Comment:
In general I think having something typed as either `false | number` isn't the best approach if we can avoid it. It's a bit confusing, but no need to change it right now.

Reply:
I agree, I can talk to Romain about this, he suggested this input when only freq power analysis was available.  Now that we are implementing Bayesian, I can see if we can simplify.

Reply:
In general I try to avoid combinations of types that are not possible to make sure we tighten the type descriptions and don't have to make assumptions about the data. Typically here, I'd like to avoid something like:
```typescript
sequentialTestingEnabled: boolean;
sequentialTestingParam: number;
```
I'm open to anything that can combine the two. Classic JS is a `nullable` number. We could also have `"disabled" | number`.
```